ui/raidboss/data/06-ew/raid/p2n.ts

```diff
@@ -0,0 +1,206 @@
+            return 'body west, head east';
+          if (bodyHeading === 'south')
+            return 'body south, head north';
+          if (bodyHeading === 'east')
+            return 'body east, head west';
+        }
+        if (matches.id === '67F7') {
+          if (bodyHeading === 'north')
+            return 'body north, head east';
+          if (bodyHeading === 'west')
+            return 'body west, head north';
+          if (bodyHeading === 'south')
+            return 'body south, head west';
+          if (bodyHeading === 'east')
+            return 'body east, head south';
+        }
+        if (matches.id === '67F9') {
+          if (bodyHeading === 'north')
+            return 'body north, head west';
+          if (bodyHeading === 'west')
+            return 'body west, head south';
+          if (bodyHeading === 'south')
+            return 'body south, head east';
+          if (bodyHeading === 'east')
+            return 'body east, head north';
+        }
+      },
+      outputStrings: {
+        text: {
+          en: 'Tank Laser on YOU',
+          de: 'Tank Laser auf DIR',
+          fr: 'Tank laser sur VOUS',
+          ja: '自分にタンクレーザー',
+          cn: '坦克射线点名',
+          ko: '탱 레이저 대상자',
+        },
+      },
+    },
+    {
+      id: 'P2N Sewage Deluge',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67F6', source: 'Hippokampos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Aoe get on grid',
+        },
+      },
+    },
+    {
+      // Spread aoe marker on some players, not all
+      id: 'P2N Tainted Flood',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6809', source: 'Hippokampos', capture: true }),
+      condition: (data, matches) => matches.target === data.me,
+      response: Responses.spread(),
+    },
+    {
+      // Drops aoe zones beneath you -> run to dodge (on everyone)
+      // Not working currently properly -> On you not working.
+      id: 'P2N Sewage Erruption',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '680D', source: 'Hippokampos', capture: false }),
+      suppressSeconds: 5,
+      response: Responses.spread(),
+    },
+    {
+      // Shortly After Sewage Erruption, not every Sewage Erruption but every Predatory Sight, maybe one trigger for both
+      id: 'P2N Predatory Sight',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '680A', source: 'Hippokampos', capture: false }),
+      response: Responses.doritoStack(),
+    },
+    {
+      // Raidwide knockback -> dont get knocked into slurry
+      id: 'P2N Shockwave',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6807', source: 'Hippokampos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Knockback stay on grid',
```

Comment:
```suggestion
          en: 'Knockback--Stay on grid',
```

Same as the other, feel free to ignore.

Reply:
Fixed in 2fe82019d5146ac7edffbd41251b7b6aaca4113d.